### ISSUE TITLE
The width of the cover in the comic list is configurable

### ISSUE THREAD

**Issue by mcpierce** [mcpierce/comixed#48]
Depends on #43 

As a comic reader, I want to control the width of the comic cover displayed to show more comics or more details. A slider is used to control the width, and the covers adjust in response to changes in the slider value. The value is persisted when the user stops changing it..

## Acceptance Criteria
1. By default, a value of 256 is used to control the width of the comic covers.
1. When the user changes the preferred height, the covers displayed adjust automatically to the specified value.
1. A record is created for the user's account storing the slider's value.
1. When the user returns to the comic list page the covers are shown in the previously stored width.

**Comment by mcpierce** [comment]
Closing this as a duplicate of #95, which is closed.
